### PR TITLE
[FIX] shared/vehicles.lua car names & other issues

### PR DESCRIPTION
**Pull request by SirOMGitsYOU**
## Description

This fixes a few grammatical and errors amongst the vehicles.lua to bring it back in-line and correct with the current vehicle set

## Checklist

- [x] Fixed error on ASBO shop tag
- [x] Fixed Duplicate vehicle names
- [x] Added missing Manchez
- [x] Fixed incorrect vehicle name.
- [x] Fixed boats being called under their brand name rather than the actual name of the boat
- [x] Fixed name for Ratloader

### CODE REVIEW

File: shared/vehicles.lua [hunk 1]
```diff
@@ -349,8 +349,8 @@ QBShared.Vehicles = {
         ['brand'] = 'Overflod',
         ['model'] = 'zeno',
         ['price'] = 1350000,
-        ['category'] = 'coupes',
-        ['categoryLabel'] = 'Coupes',
+        ['category'] = 'super',
+        ['categoryLabel'] = 'Super',
```
Comment (Z3rio) [inline shared/vehicles.lua:353]: A blista isnt a super car?

File: shared/vehicles.lua [hunk 2]
```diff
@@ -946,6 +946,16 @@ QBShared.Vehicles = {
...
+    ['manchez3'] = {
+        ['name'] = 'Manchez Scout C',
+        ['brand'] = 'Maibatsu',
+        ['model'] = 'manchez3',
+        ['price'] = 14000,
+        ['category'] = 'motorcycles',
+        ['categoryLabel'] = 'Motorcycles',
+        ['hash'] = `manchez3`,
+        ['shop'] = 'pdm',
+    },
```
Comment (Z3rio) [inline shared/vehicles.lua:958]: This motorcycle doesnt exist when testing with the newest game build

**Review by Z3rio — CHANGES_REQUESTED** [review]

File: shared/vehicles.lua (hunk 1 above)
Comment (SirOMGitsYOU) [reply shared/vehicles.lua:353]: image
thats not for the blista

**Comment by Z3rio** [comment]
Is this ready to be comitted or are you still working on it?

File: shared/vehicles.lua [hunk 3]
```diff
@@ -4487,7 +4497,7 @@ QBShared.Vehicles = {
         ['shop'] = 'boats',
     },
     ['seashark3'] = {
-        ['name'] = 'Seashark',
+        ['name'] = 'Seashark 3',
```
Comment (Z3rio) [inline shared/vehicles.lua:4500]:
```suggestion
        ['name'] = 'Seashark Yacht',
```

File: shared/vehicles.lua [hunk 4]
```diff
@@ -4517,8 +4527,8 @@ QBShared.Vehicles = {
         ['shop'] = 'boats',
     },
     ['tropic2'] = {
-        ['name'] = 'Shitzu',
-        ['brand'] = 'Tropic',
+        ['name'] = 'Tropic 2',
```
Comment (Z3rio) [inline shared/vehicles.lua:4530]:
```suggestion
        ['name'] = 'Tropic Yacht',
```

File: shared/vehicles.lua [hunk 5]
```diff
@@ -4547,8 +4557,8 @@ QBShared.Vehicles = {
         ['shop'] = 'boats',
     },
     ['dinghy3'] = {
-        ['name'] = 'Nagasaki',
-        ['brand'] = 'Dinghy',
+        ['name'] = 'Dinghy 3',
```
Comment (Z3rio) [inline shared/vehicles.lua:4560]:
```suggestion
        ['name'] = 'Dinghy (Heist)',
```

File: shared/vehicles.lua [hunk 6]
```diff
@@ -4557,8 +4567,8 @@ QBShared.Vehicles = {
         ['shop'] = 'boats',
     },
     ['dinghy4'] = {
-        ['name'] = 'Nagasaki',
-        ['brand'] = 'Dinghy',
+        ['name'] = 'Dinghy 4',
```
Comment (Z3rio) [inline shared/vehicles.lua:4570]:
```suggestion
        ['name'] = 'Dinghy Yacht',
```

File: shared/vehicles.lua [hunk 7]
```diff
@@ -4587,7 +4597,7 @@ QBShared.Vehicles = {
         ['shop'] = 'boats',
     },
     ['speeder2'] = {
-        ['name'] = 'Speeder',
+        ['name'] = 'Speeder 2',
```
Comment (Z3rio) [inline shared/vehicles.lua:4600]:
```suggestion
        ['name'] = 'Speeder Yacht',
```

File: shared/vehicles.lua [hunk 8]
```diff
@@ -4617,7 +4627,7 @@ QBShared.Vehicles = {
         ['shop'] = 'boats',
     },
     ['toro2'] = {
-        ['name'] = 'Toro',
+        ['name'] = 'Toro II',
```
Comment (Z3rio) [inline shared/vehicles.lua:4630]:
```suggestion
        ['name'] = 'Toro Yacht',
```

**Review by Z3rio — CHANGES_REQUESTED** [review]
It looks alot better to use the actual names instead of putting numbers in the names.
You can see the actual names for these vehicles through this website: https://gtacars.net/gta5

**Comment by SirOMGitsYOU** [comment]
> It looks alot better to use the actual names instead of putting numbers in the names. You can see the actual names for these vehicles through this website: https://gtacars.net/gta5

Thank you - I will use this website, the gtabase one didn't have any additional names for those ones so I just labeled them with numbers :)

**Review by Z3rio — APPROVED** [review]

**Pull request merged**